src/main/java/com/adyen/service/balanceplatform/BalanceAccounts.java

```diff
@@ -36,11 +36,11 @@ public BalanceSweepConfigurationsResponse listSweeps(String balanceAccountId, Ma
         return BalanceSweepConfigurationsResponse.fromJson(jsonResult);
     }
 
-    public BalanceSweepConfigurationsResponse createSweep(String balanceAccountId, SweepConfigurationV2 request) throws IOException, ApiException {
+    public SweepConfigurationV2 createSweep(String balanceAccountId, SweepConfigurationV2 request) throws IOException, ApiException {
```

Comment:
Will we have to modify this service again when this object name gets fixed?

Reply:
Nah I think this is the actual name, as that class has been around for a while.